app/controllers/episodes_controller.rb

```diff
@@ -28,10 +30,21 @@ def update
     end
   end
 
+  def add_start
+    @episode.update(actual_start: Time.now)
+    redirect_to episode_path(@episode)
+  end
+
+  def add_finish
+    @episode.update(actual_finish: Time.now)
+    redirect_to episode_path(@episode)
+  end
+
   private
 
   def set_episode
     @episode = Episode.find(params[:id])
+    @episode.update_youtube_info
```

Comment:
Nope, you can't update something here. And why we need to make it every time You call this controller?

Reply:
I don't know where to update because statuses aren't merged yet. I have brought back button but with proper root.